src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -86,7 +86,7 @@ def drop_application(self, auto_yes: bool):
                     )
                 )
                 if not should_drop_object:
-                    print(f"Did not drop application {self.app_name}.")
+                    cc.step(f"Did not drop application {self.app_name}.")
```

Comment:
This is neither a step nor a warning. Maybe we need to consider adding some kind of message API for cases like this. For now it could be an alias for step, but conceptually it would keep the presentation decisions centralized in the SnowCLI core. Thoughts?

Alternately, we use warning for now.

Reply:
Same as above, using `step` because the user rejected the drop and now we're just informing them that we did not drop. Though this also falls under the category of a simple information message imo, and is neither a step nor a warning (like you mentioned).